cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -118,21 +120,151 @@ def upgrade_5_0_0(session_key, logger):
     except:
         logger.info("Old macro value for (cs_authentication_vpn_login_attemps_outside_working_hour_filter) in the user environment does not exist, skipping the upgrade step.")
 
-    # TODO - Upgrade step to call to the rest endpoint - CyencesProductConfiguration/product_config - to automatically enable/disable all the renamed alerts based on existing settings
+    try:
+        enabled_product = conf_manager.get_conf_stanza("cs_configurations", "product_config")[0]["content"].get("enabled_products")
+        product = enabled_product.split(",")[0].strip()
+
+        if product:
+            payload = {
+                "product": product,
+                "enabled": 0,
+            }
+            rest.simpleRequest(
+                "/servicesNS/nobody/{}/CyencesProductConfiguration/product_config?output_mode=json".format(
+                    cs_utils.APP_NAME),
+                postargs={"data": json.dumps(payload)},
+                method="POST",
+                sessionKey=session_key,
+                raiseAllErrors=True,
+            )
+
+            logger.info("Disabled the product={}".format(product))
+
+            payload = {
+                "product": product,
+                "enabled": 1,
+            }
+
+            rest.simpleRequest(
+                "/servicesNS/nobody/{}/CyencesProductConfiguration/product_config?output_mode=json".format(
+                    cs_utils.APP_NAME
+                ),
+                postargs={"data": json.dumps(payload)},
+                method="POST",
+                sessionKey=session_key,
+                raiseAllErrors=True,
+            )
+
+            logger.info("Re-enabled the product={}".format(product))
+    except Exception:
+        logger.info("Error while product enable/disable. Please manually disable/enable the product on Cyences App Configuration > Product Setup page")
+
+    alerts_to_disable = [
```

Comment:
Add comment on why we are doing this:

Removed alert might be present in apps/cyences_app_for_splunk/local/savedsearches.conf, which would constantly generate errors or run searches in all times, etc, etc. To avoid this we are disabling all these alerts if they present in the local folder.